demo/combo-box-basic-demos.html

```diff
@@ -6,91 +6,155 @@
+    <h3>Clear button</h3>
+    <vaadin-demo-snippet id="clear-button" when-defined="vaadin-combo-box">
       <template preserve-content>
-        <vaadin-combo-box label="Element" clear-button-visible></vaadin-combo-box>
+        <vaadin-combo-box label="Label" clear-button-visible></vaadin-combo-box>
         <script>
-          window.addDemoReadyListener('#combo-box-basic-demos-clear-button', function(document) {
+          window.addDemoReadyListener('#clear-button', function(document) {
             const comboBox = document.querySelector('vaadin-combo-box');
-            comboBox.items = ['Hydrogen', 'Helium', 'Lithium'];
-            comboBox.value = 'Hydrogen';
+            comboBox.items = ['Option one', 'Option two'];
+            comboBox.value = 'Option one';
           });
         </script>
       </template>
     </vaadin-demo-snippet>
 
+    <h3>Value change event</h3>
+    <vaadin-demo-snippet id="allow-custom-value" when-defined="vaadin-combo-box">
+      <template preserve-content>
+        <vaadin-combo-box label="Label"></vaadin-combo-box>
+        <div id="output">Selected value:</div>
+        <script>
+          window.addDemoReadyListener('#allow-custom-value', function(document) {
+            const comboBox = document.querySelector('vaadin-combo-box');
+            const output = document.querySelector('#output');
+            comboBox.items = ['Option one', 'Option two'];
+            comboBox.addEventListener('change', function(event) {
+              output.textContent = event.target.value;
+            });
+          });
+        </script>
+      </template>
+    </vaadin-demo-snippet>
 
-    <h3>Allow Custom Values</h3>
+    <h3>Allow custom values</h3>
     <p>Allow the user to set any value for the field in addition to selecting a value from the dropdown menu.</p>
     <vaadin-demo-snippet id="combo-box-basic-demos-allow-custom-values" when-defined="vaadin-combo-box">
       <template preserve-content>
-        <vaadin-combo-box label="Element" allow-custom-value></vaadin-combo-box>
-        <p>Selected value: <span></span></p>
-
+        <vaadin-combo-box label="Fruit" allow-custom-value></vaadin-combo-box>
+        <p>Selected value: <span></span></span></p>
         <script>
           window.addDemoReadyListener('#combo-box-basic-demos-allow-custom-values', function(document) {
             const comboBox = document.querySelector('vaadin-combo-box');
             const span = document.querySelector('span');
+            const fruits = ['Apple', 'Orange', 'Banana'];
+            comboBox.items = fruits;
+            comboBox.value = span.textContent = 'Apple';
+
+            comboBox.addEventListener('change', function() {
+              span.textContent = comboBox.value;
+            });
+          });
+        </script>
+      </template>
+    </vaadin-demo-snippet>
+
+    <h3>Storing custom values</h3>
+    <vaadin-demo-snippet id="combo-box-second-custom-value" when-defined="vaadin-combo-box">
```

Comment:
This could be something more descriptive than "second-custom-value", maybe "storing-custom-values" as the title says.

Reply:
Done